README.md

```diff
@@ -1,25 +1,25 @@
-# Software Supply Chain Attribute Integrity (SCAI) Generator
+# in-toto SCAI Generator and Demos
 
 The Software Supply Chain Attribute Integrity, or SCAI (pronounced "sky"),
-specification is a succint data format for claims and evidence about
+framework is a succint data format specification for claims and evidence about
 attributes and integrity about a software artifact and its supply chain.
 
 For more details read our [intro doc] or the full [SCAI spec doc].
 
 ## In this repo
 
-This repo provides CLI tools for automatically generating valid,
-in-toto compatible SCAI metadata, and for checking the generated
-metadata against a given policy.
+This repo provides [Go](scai-gen/) and [Python](python/) implementations of
+CLI tools for automatically generating valid, in-toto ITE-6 compliant SCAI
```

Comment:
I think we ought to stop referring to the framework as ITE-6, and just directly refer to the attestation framework.